### PR TITLE
FixBug:  Load data Is error in json load

### PR DESCRIPTION
**Pull request by worker24h**
The data  is always 1 by json load
https://github.com/apache/incubator-doris/issues/3697

### CODE REVIEW

File: be/src/exprs/json_functions.h [hunk 1]
```diff
@@ -72,6 +72,7 @@ class JsonFunctions {
 
     /**
      * The `document` parameter must be has parsed.
+     * return Value Is Array object
```
Comment (morningman) [inline be/src/exprs/json_functions.h:75]: better to change its name to `get_json_array_from_parsed_json()` to make it more explict.

And in `src//exec/json_scanner.cpp` line 270, you can this method, but judge the return value as either array or non-array? the `else` should not happen?

**Review by morningman — CHANGES_REQUESTED** [review]

File: be/src/exprs/json_functions.h (hunk 1 above)
Comment (worker24h) [reply be/src/exprs/json_functions.h:75]: The method MUST return ArrayObject.
 `else if (!root->IsArray())` is true, then I warp a ArrayObject  in `root` outside.
`else` case  MUST be ArrayObject

Comment (worker24h) [reply be/src/exprs/json_functions.h:75]: The method name changed to `get_json_objects_from_parsed_json` ??

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**